command/init.go

```diff
@@ -50,6 +51,15 @@ func (c *InitCommand) Run(args []string) int {
 		RecoveryPGPKeys:   recoveryPgpKeys,
 	}
 
+	switch len(rootTokenPgpKey) {
+	case 0:
+	case 1:
+		initRequest.RootTokenPGPKey = rootTokenPgpKey[0]
```

Comment:
There are no headers -- this uses the raw (base64 encoded) key bytes. It's in a format corresponding to PGP but not much reason to parse ahead of time since we also need to parse and check on the server.